parser.y

```diff
@@ -1426,6 +1440,25 @@ AlterTableSpec:
 		parser.lastErrorAsWarn()
 	}
 
+WithValidationOpt:
```

Comment:
return `true` or `false`?

Reply:
validation could have default value, in which case restore should omit it

Reply:
fine